public/pages/CreateDetector/components/ConfigureFieldMapping/components/RequiredFieldMapping/FieldNameSelector.tsx

```diff
@@ -15,43 +14,54 @@ interface SIEMFieldNameProps {
 }
 
 interface SIEMFieldNameState {
-  selectedOption?: string;
+  selectedOptions: EuiComboBoxOptionOption<string>[];
   errorMessage?: string;
 }
 
 export default class FieldNameSelector extends Component<SIEMFieldNameProps, SIEMFieldNameState> {
   constructor(props: SIEMFieldNameProps) {
     super(props);
+
+    let selectedOptions = [] as EuiComboBoxOptionOption<string>[];
+    if (props.selectedField) {
```

Comment:
Club this with delcaration?
```
const selectedOptions = props.selectedField 
? [ { label: ... } ] 
: [];
```

Reply:
Done. Mixed with the comment above:
`const selectedOptions: EuiComboBoxOptionOption<string>[] = props.selectedField
      ? [{ label: props.selectedField }]
      : [];`